components/match_table/commons/match_table.lua

```diff
@@ -0,0 +1,753 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MatchTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local DateExt = require('Module:Date/Ext')
+local Game = require('Module:Game')
+local LeagueIcon = require('Module:LeagueIcon')
+local Logic = require('Module:Logic')
+local Lpdb = require('Module:Lpdb')
+local Lua = require('Module:Lua')
+local Math = require('Module:MathUtil')
+local Page = require('Module:Page')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Team = require('Module:Team')
+local Tier = require('Module:Tier/Custom')
+local VodLink = require('Module:VodLink')
+
+local PlayerExt = Lua.import('Module:Player/Ext')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local PLAYER_MODE = 'player'
```

Comment:
Shouldn't we stick to the term `solo` here?
Could use Opponent.MODE directly then.

Reply:
hmm
my thinking was that we also get non solo results for a player input

Reply:
You could still do that, no?
I guess it's debatable, but using opponent modes would also offer the possibility to later extend to other modes (if that is a thing), and also be more open to non-esports-wikis.